packages/react/src/components/MultiSelect/FilterableMultiSelect.js

```diff
@@ -398,9 +398,9 @@ export default class FilterableMultiSelect extends React.Component {
                       aria-controls={`${id}__menu`}
                       aria-autocomplete="list"
                       ref={(el) => (this.inputNode = el)}
+                      id={`${id}-input`}
```

Comment:
Might make sense for the passed in `id` to be on the `<input>` instead of the field since folks seem to use it to target the input in selenium-based tests 👀